### PR TITLE
User can include a state in a winning combination

### PR DESCRIPTION
**Pull request by kennykimjr**
screen shot 2017-06-13 at 2 34 11 pm

### CODE REVIEW

File: data.js [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+    name: 'WA',
+    ev: 12
+  },
+  {
+    name: 'WV',
+    ev: 5
+  },
+  {
+    name: 'WI',
+    ev: 10
```
Comment (victormejia) [inline data.js]: don't be afraid to use `votes` or `electoralVotes`. Someone new to the project may wonder what is `ev`?

File: index.html [hunk 2]
```diff
@@ -10,7 +10,10 @@
       <h1>Election Winner Project</h1>
       <h2>Create a combination of states to win 270 electoral votes</h2>
     </div>
-    <img class="usa" src="http://www.theharvesteronline.com/congregations/images/usa.png"/>
-    <script type="text/javascript" src="main.js"></script>
+    <div class="ocean">
+      <object class="map" data="usamap.svg" type="image/svg+xml" id="usa"></object>
+      <script type="text/javascript" src="data.js"></script>
+      <script type="text/javascript" src="main.js"></script>
```
Comment (victormejia) [inline index.html]: script tags should be at the end of the body

File: main.js [hunk 3]
```diff
@@ -1,5 +1,40 @@
-var usa = document.getElementsByClassName('usa')[0]
+// In this directory in order for this to work you need to type
+// type into the console http-server
+// Use the first link given
+// npm js http-server
+// file:///Users/kennykim/electionwinner/index.html
+// open -n -a /Applications/Google\ Chrome.app --args --user-data-dir="/tmp/someFolderName" --disable-web-security
+// var usa = document.querySelector('map').getSVGDocument().getElementById('AK').setAttribute('fill', 'red')
```
Comment (victormejia) [inline main.js]: what are all these instructions? If you need setup instructions, you need to include those in a README file.  Read up on npm scripts: https://css-tricks.com/why-npm-scripts/

File: main.js [hunk 4]
```diff
@@ -1,5 +1,34 @@
-var usa = document.getElementsByClassName('usa')[0]
+// open -n -a /Applications/Google\ Chrome.app --args --user-data-dir="/tmp/someFolderName" --disable-web-security
```
Comment (victormejia) [inline main.js]: why do I need this?

File: main.js [hunk 5]
```diff
@@ -1,5 +1,34 @@
...
+
+usa.addEventListener('load', function () {
+  var statesSVG = usa.getSVGDocument()
+  for (var i = 0; i < stateVals.length; i++) {
+    stateVals[i].svgdata = statesSVG.getElementById(stateVals[i].name)
+    stateVals[i].svgdata.addEventListener('click', clickedState(i))
+  }
+  function clickedState(i) {
+    return function () {
+      console.log(stateVals[i].svgdata.style.fill)
```
Comment (victormejia) [inline main.js]: remove any console logging

File: main.js [hunk 6]
```diff
@@ -1,5 +1,34 @@
...
+  else if (color === 'gold') {
+    return ''
+  }
+}
+
+usa.addEventListener('load', function () {
+  var statesSVG = usa.getSVGDocument()
+  for (var i = 0; i < stateVals.length; i++) {
+    stateVals[i].svgdata = statesSVG.getElementById(stateVals[i].name)
+    stateVals[i].svgdata.addEventListener('click', clickedState(i))
```
Comment (victormejia) [inline main.js]: you're adding 50 event listeners here. Try to add only one event listener, and decide what to do based on the `event.target`

File: main.js (hunk 4 above)
Comment (kennykimjr) [inline main.js]: Apparently, SVG's can't be accessed without disabling the same-origin policy. I'll elaborate upon this in the readme.

Comment (victormejia) [inline main.js]: What you actually need is to run this on a local server. Checkout https://github.com/indexzero/http-server, and create an npm script that will start a server

**Comment by victormejia** [comment]
@kennykimjr no need to include the issue number in the PR title.

**Comment by victormejia** [comment]
update screenshot also :)

**Comment by kennykimjr** [comment]
screen shot 2017-06-13 at 4 36 09 pm

**Pull request merged**